tests/Doctrine/Tests/Mocks/CacheKeyMock.php

```diff
@@ -13,9 +13,9 @@
  */
 class CacheKeyMock extends CacheKey
 {
-    /** @param string $hash The string hash that represend this cache key */
+    /** @param string $hash The string hash that represents this cache key */
     public function __construct(string $hash)
     {
-        $this->hash = $hash;
+        parent::__construct($hash);
     }
```

Comment:
:thinking: Can't you remove the constructor entirely instead? Or maybe even the whole class is useless?

Reply:
Actually we would need a test with constructor and without, because despite the deprecation both ways need to work.

Reply:
> 🤔 Can't you remove the constructor entirely instead?

I can, but I was reluctant to do so because it's stricter than `CacheKey`'s.

> Or maybe even the whole class is useless?

`CacheKey` is abstract and I don't really want to change that. So if we dropeed `CacheKeyMock`, I would need to replace it with an anonymous class in every test that uses it. I'm not sure that's an improvement really.

Reply:
> Actually we would need a test with constructor and without, because despite the deprecation both ways need to work.

I've pushed two new tests.

Reply:
Didn't notice the extra strictness 👍